### PR TITLE
add revoke cmd

### LINKED ISSUE #30: Add revoke command.
I want to revoke role from user.

```sh
bqiam revoke dataset WRITER -p project -u aaa@example.com -d dataset
```

### PR DESCRIPTION
**Pull request by hirosassa**
fixes https://github.com/hirosassa/bqiam/issues/30

### CODE REVIEW

**Comment by hirosassa** [comment]
@snowhork @kitagry I added revoke permission commands. plz review.

File: bqrole/project.go [hunk 1]
```diff
@@ -66,6 +67,49 @@ func PermitProject(role, project string, users []string, yes bool) error {
...
+		reader := bufio.NewReader(os.Stdin)
+		res, err := reader.ReadString('\n')
+
+		if err != nil || strings.TrimSpace(res) != "y" {
+			fmt.Println("Abort.")
+			return nil
+		}
+	}
+
+	defer client.Close()
```
Comment (kitagry) [inline bqrole/project.go]: I think client.Close should be called immediately

```go
	client, err := bq.NewClient(ctx, project)
	if err != nil {
		return errors.New("failed to create bigquery Client")
	}
	defer client.Close()
```

**Comment by kitagry** [comment]
LGTM

File: bqrole/project.go (hunk 1 above)
Comment (hirosassa) [reply bqrole/project.go]: Thanks! fixed

**Comment by snowhork** [comment]
LGTM!

**Pull request merged**